### PR TITLE
feat(discover): Calculate row heights

### PR DESCRIPTION
**Pull request by lynnagara**
Calculate row heights dynamically based on content of cells in that row.
Max row height is 3 rows of text.

### CODE REVIEW

File: src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx [hunk 1]
```diff
@@ -168,13 +190,24 @@ export default class ResultTable extends React.Component {
     // Add one column at the end to make sure table spans full width
     const colCount = cols.length + (showEventLinks ? 1 : 0) + 1;
 
-    const maxVisibleResults = Math.min(data.length, 10);
+    const maxVisibleResults = Math.min(data.length, 12);
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx]: Should `12` be a constant here?

File: src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx [hunk 2]
```diff
@@ -135,6 +136,27 @@ export default class ResultTable extends React.Component {
...
+      return this.measureText(getDisplayText(row[col]), false);
+    });
+    const maxColWidth = Math.max(...colWidths, 0);
+
+    const rows = Math.max(
+      Math.min(Math.ceil(maxColWidth / (MAX_COL_WIDTH - CELL_PADDING)), 3),
+      1
+    );
+
+    return TABLE_ROW_HEIGHT * rows + 1; // 1px for border;
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx]: Should `1` be a constant?

File: src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx [hunk 3]
```diff
@@ -135,6 +136,27 @@ export default class ResultTable extends React.Component {
...
+    }
+
+    const row = data[rowIndex - 1]; // -1 offset due to header row
+    const colWidths = columnsToCheck.map(col => {
+      return this.measureText(getDisplayText(row[col]), false);
+    });
+    const maxColWidth = Math.max(...colWidths, 0);
+
+    const rows = Math.max(
+      Math.min(Math.ceil(maxColWidth / (MAX_COL_WIDTH - CELL_PADDING)), 3),
```
Comment (billyvg) [inline src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx]: Can you add a comment for what this calc is for?

Comment (lynnagara) [reply src/sentry/static/sentry/app/views/organizationDiscover/result/table.jsx]: added

**Review by billyvg — APPROVED** [review]

**Pull request merged**